packages/firestore/__tests__/firestore.test.ts

```diff
@@ -691,30 +714,6 @@ describe('Firestore', function () {
     });
   });
 
-  describe('FirestorePersistentCacheIndexManager', function () {
-    it('is exposed to end user', function () {
-      const firestore1 = firebase.firestore();
-      firestore1.settings({ persistence: true });
-      const indexManager = firestore1.persistentCacheIndexManager();
-      expect(indexManager).toBeDefined();
-      expect(indexManager!.constructor.name).toEqual('FirestorePersistentCacheIndexManager');
-
-      expect(indexManager!.enableIndexAutoCreation).toBeInstanceOf(Function);
-      expect(indexManager!.disableIndexAutoCreation).toBeInstanceOf(Function);
-      expect(indexManager!.deleteAllIndexes).toBeInstanceOf(Function);
-
-      const firestore2 = firebase.firestore();
-      firestore2.settings({ persistence: false });
-
-      const nullIndexManager = firestore2.persistentCacheIndexManager();
-
-      expect(nullIndexManager).toBeNull();
-
-      const nullIndexManagerModular = getPersistentCacheIndexManager(firestore2);
-      expect(nullIndexManagerModular).toBeNull();
-    });
-  });
```

Comment:
This one mixed modular and non-modular tests, so I split it and moved the chunks to the right spots